plugins/services/src/js/service-configuration/ServiceConfigBaseSectionDisplay.js

```diff
@@ -91,7 +98,13 @@ class ServiceConfigBaseSectionDisplay extends React.Component {
 
     return (
       <ConfigurationMapSection>
-        {configurationMapRows}
+        <MountService.Mount
+          appConfig={this.props.appConfig}
+          onEditClick={onEditClick}
+          type={this.getMountType()}
+        >
+          <div>{configurationMapRows}</div>
```

Comment:
Would you please write a few words why `<div>` is needed, so everyone is onboard?

Reply:
it isn't. 😆  Just forgot to remove it… the Mountservice will wrap multiple Elements itself via `ReactUtil.wrapElements`